### ISSUE TITLE
Invalid \escape on channels with special characters in name

### ISSUE THREAD

**Issue by christianboyle** [NotJoeMartinez/yt-fts#8]
Things seem to work as expected in testing on this channel: https://www.youtube.com/@CodexCommunity/videos

But trying this channel results in an error: https://www.youtube.com/@PerfectGuyLife/videos

```json.decoder.JSONDecodeError: Invalid \escape: line 1 column 212 (char 211)```

My guess is that the 2nd channel uses emoji in some of their titles, like ```🔴 Fishtank is LIVE 🐟 50% OFF Weekend```

Any thoughts on this?

**Comment by NotJoeMartinez** [comment]
Problem on is on line 210 in the `get_channel_name` function. 
```python3
script = soup.find('script', type='application/ld+json')
data = json.loads(script.string)
```
 for some reason `json.loads()` is breaking when it tries to read the channel name from the string:
 ```
 "name": "Sam \x26 Nick\x27s Perfect Clips"
 ```

**Comment by NotJoeMartinez** [comment]
Fixed with 29e0f11db0ea3b0b1c1e1c7e491d5faa38d2b602

**Comment by christianboyle** [comment]
@NotJoeMartinez Nice, thanks for the quick fix!
